cypress/fixtures/docs/dynamicObjects.md

```diff
@@ -0,0 +1,275 @@
+# Dynamic Objects (Only Supported in JS Objects)
+
+## [Supported Dynamic Glue Codes](../../support/step_definitions/dynamicCalls.md)
+
```

Comment:
To use dynamic firrebolt objects, we need to use dynamic glue codes listed here - Link here

Reply:
done